integration_tests/pages/prisons/categoryGroups/addCategoryGroup.ts

```diff
@@ -10,7 +10,7 @@ export default class AddCategoryGroupPage extends Page {
   }
 
   selectLevel = (level: string): void => {
-    cy.get(`[name="prisonerCategories"][value="${level}"]`).check()
+    cy.get(`[name="prisonerCategories"][value="${level}"]`).check({ force: true })
```

Comment:
govuk-frontend v5.1.0 has changed something with checkbox input/labels that makes Cypress think a element is hidden when it is not. Fix to force the .check()